src/components/SpaceProposalPage.vue

```diff
@@ -277,5 +278,17 @@ onMounted(() => {
       :selected-choices="selectedChoices"
       @close="isModalPostVoteOpen = false"
     />
+    <SpaceProposalVotesModal
```

Comment:
This should be inside `SpaceProposalVotesList` and you should use all the sorting login from there instead of repeating is inside the modal again

Reply:
If we want to do it in SpaceProposalVotesList where we have only 10 votes, then we create there another array of votes for searching.

Reply:
Hm yes good point, but we should try to always reuse code where possible.

Reply:
Maybe a composable would make sense

Reply:
When do you start reuse code? With >1 pieces of duplication code or >=3?

Reply:
>1

Reply:
Specially if it's larger amount of code